billing-service/src/main/java/com/petclinic/billing/datalayer/Bill.java

```diff
@@ -27,7 +30,9 @@
 
     public Bill(){}
 
-    public Bill(int billId, Date date, String visitType, double amount){
+    //int customerId ---> Add this to constructor maybe?
```

Comment:
I think we can remove the comment out here and implement the field your considering. Do you agree?